statsd-client/dogstatsd.ml

```diff
@@ -70,7 +73,7 @@ module ServiceCheck = struct
       [ `Ok
       | `Warning
       | `Critical
-      | `Unknown ]
+      | `Unknown ] [@@deriving sexp]
```

Comment:
It's generally nice to put the `[@@deriving ...]` on the next line for multi-line types